pages/404.tsx

```diff
@@ -0,0 +1,121 @@
+import { Heading, Text } from "@telegraph/typography";
+import { Box } from "@telegraph/layout";
+import { Page } from "@/components/ui/Page";
+import Meta from "@/components/Meta";
+import { useRouter } from "next/router";
+import { useEffect, useState } from "react";
+import Link from "next/link";
+import { motion } from "framer-motion";
+import { MenuItem } from "@telegraph/menu";
+import { PLATFORM_SIDEBAR } from "@/data/sidebars/platformSidebar";
+
+/**
+ * Not seeing 404 page in development? Please read!
+ * First, this works when you build. This is only a development issue.
```

Comment:
You're talking about local dev right? May want to specify! Great explanation tho ty
```suggestion
 * First, the 404 page does load when you build. This is only a local development issue.
```